lib/core/text/abstract_text.nit

```diff
@@ -2021,7 +2021,7 @@ end
 # see `alpha_comparator`
 private class AlphaComparator
 	super Comparator
-	redef fun compare(a, b) do return a.to_s <=> b.to_s
+	redef fun compare(a, b) do return a.as(not null).to_s <=> b.as(not null).to_s
```

Comment:
Maybe allow to compare arrays that contains null ?

~~~nit
if a == b then return 0
if a == null then return -1
if b == null then return 1
return a <=> b
~~~